src/components/Authenticator.tsx

```diff
@@ -63,7 +69,22 @@ export function AuthenticationEnsurer ({
   children: JSX.Element | JSX.Element[]
 }): JSX.Element {
   const [{ submitted, accounts, client }] = useAuthenticator()
+  const plausible = usePlausible()
   const authenticated = !!accounts.length
+  const previousAuth = useRef<boolean>(authenticated)
+
+  useEffect(() => {
+    console.debug('auth changed:', {
+      was: previousAuth.current,
+      now: authenticated
+    })
+    // Only track if the transition is from unauthenticated ➝ authenticated
+    if (!previousAuth.current && authenticated) {
+      plausible('Login Successful')
```

Comment:
I think this might track a login event when the user is logged in but they freshly visit the site or just refresh the page...

Reply:
yes it does fire on fresh start and page refresh. however i could not pass events from Authenticator.tsx in the ui-react components as it uses Fragments, would need some workaround. happy for someone to take this over.